op-guide: update tidb-ansible deployment (#1001)
pingcap/docs: op-guide/ansible-deployment.md

## Step 3: Download TiDB-Ansible to the Control Machine

1. Log in to the Control Machine using the `tidb` user account and enter the `/home/tidb` directory. The relationship between `tidb-ansible` and TiDB versions is as follows:

    | TiDB version | tidb-ansible tag | Note |
    | -------- | ---------------- | --- |
    | 2.0 version | v2.0.10, v2.0.11 | It is the latest 2.0 stable version which can be used in the production environment. |
    | 2.1 version | v2.1.1 ~ v2.1.6 | It is the latest 2.1 stable version which can be used in the production environment (recommended). |
    | 3.0 version | v3.0.0-beta, v3.0.0-beta.1 | It is currently a beta version which is not recommended to use in the production environment. |

>     | The latest version | None | It includes the newest features and is updated on a daily basis, so it is not recommended to use it in the production environment. |

Reviewer: ```suggestion
    | `master` branch | None | It includes the newest features and is updated on a daily basis, so it is not recommended to use it in the production environment. |
```
In the previous text it was called 'master'. I think we should keep using that name because it has meaning in the code (as does release-2.0, release-2.1). Calling it 'latest' does not convey the same meaning.

Author: @liubo0127 @lilin90 How do you think? Should I change it back?

Author: Feedback addressed. PTAL again. @morgo @lilin90

Reviewer 2: Let's keep using `master`.

2. Download the corresponding TiDB-Ansible versions from the [TiDB-Ansible project](https://github.com/pingcap/tidb-ansible). The default folder name is `tidb-ansible`.

    > **Note:** You should use the corresponding tidb-ansible version when you deploy and upgrade the TiDB cluster. Errors might occur when you use multiple versions by modifying the `inventory.ini` file.
    

>     - Download the tidb-ansible with the specified tag:

Reviewer 2: ```suggestion
    - Download the tidb-ansible with a specified tag:
```

    
        ```
        $ git clone -b $tag https://github.com/pingcap/tidb-ansible.git
        ```

>     - Download the tidb-ansible that corresponds to the latest version:

Reviewer 2: ```suggestion
    - Download the tidb-ansible that corresponds to the latest TiDB version:
```
